src/plotman/_tests/log_parser_test.py

```diff
@@ -0,0 +1,54 @@
+import importlib.resources
+
+from plotman._tests import resources
+from plotman.log_parser import PlotLogParser
+
+def test_should_correctly_parse():
+  with importlib.resources.path(resources, "2021-04-04T19_00_47.681088-0400.log") as p:
+    logfile_path = p
+
+  parser = PlotLogParser()
+  info = parser.parse(logfile_path)
+
+  assert info.plot_id == "3eb8a37981de1cc76187a36ed947ab4307943cf92967a7e166841186c7899e24"
```

Comment:
Maybe create a reference `PlotInfo` instance and assert equality to that?  That guarantees the complete object is compared whereas here it is easy to leave an attribute unchecked.